Dockerfile

```diff
@@ -1,25 +1,33 @@
-FROM debian:9 AS builder
+FROM buildpack-deps:stretch AS builder
```

Comment:
Changed the base build to buildpack-deps:stretch to pull in open ssl support. 
If we would rather leave it at debian:9 and pull in those dependencies ourself, we can try to do that as well.

Reply:
That includes a bunch of unrelated packages. I'd rather just include what we need.